python/python/bystro/supervised_ppca/ppca_augmented.py

```
@@ -713,7 +715,7 @@ def fit(
         X_dm = X - self.mean_x
         Y_dm = Y - self.mean_y
         N, self.p = X.shape
-        q = Y.shape[1]
+        p,q = X.shape[1],Y.shape[1]
```

Ruff

Local variable `p` is assigned to but never used

(F841)